Documentation/PageTsconfig/Mod/WebInfo.rst

```diff
@@ -1,109 +1,116 @@
+..  index::
+    web_info.menu.function
+    Module menu; Info
+..  _pageblindingfunctionmenuoptions-webinfo:
 
 menu.function
 =============
 
-:aspect:`Datatype`
-   array
-
-:aspect:`Description`
-   Disable elements of the "Function selector" in the document header of the module. The keys for single
-   items can be found by browsing *System > Configuration > $GLOBALS['TBE_MODULES_EXT']*.
-
-   .. figure:: /Images/ManualScreenshots/Info/FunctionMenuInfoModule.png
-      :alt: The function menu of the Info module
-
-      The function menu of the Info module
-
-   .. warning::
-
-      Blinding the function mMenu items is not hardcore access control! All it
-      does is hide the possibility of accessing that module functionality
-      from the interface. It might be possible for users to hack their way
-      around it and access the functionality anyways. You should use the
-      option of blinding elements mostly to remove otherwise distracting options.
-
-:aspect:`Example`
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/page.tsconfig
-
-      mod.web_info.menu.function {
-         # Disable item "Page Tsconfig"
-         TYPO3\CMS\Info\Controller\InfoPageTyposcriptConfigController = 0
-         # Disable item "Log"
-         TYPO3\CMS\Belog\Module\BackendLogModuleBootstrap = 0
-         # Disable item "Pagetree Overview"
-         TYPO3\CMS\Info\Controller\PageInformationController = 0
-         # Disable item "Localization Overview"
-         TYPO3\CMS\Info\Controller\TranslationStatusController = 0
-         # Disable item "Linkvalidator"
-         TYPO3\CMS\Linkvalidator\Report\LinkValidatorReport = 0
-      }
+..  confval:: menu.function
+    :name: mod-web-info-menu-function
+    :type: array
+
+    Disable elements of the "Function selector" in the document header of the module. The keys for single
+    items can be found by browsing *System > Configuration > $GLOBALS['TBE_MODULES_EXT']*.
+
+    ..  figure:: /Images/ManualScreenshots/Info/FunctionMenuInfoModule.png
+        :alt: The function menu of the Info module
+
+        The function menu of the Info module
+
+    ..  warning::
+
+        Blinding the function menu items is not hardcore access control! All it
+        does is hide the possibility of accessing that module functionality
+        from the interface. It might be possible for users to hack their way
+        around it and access the functionality anyways. You should use the
+        option of blinding elements mostly to remove otherwise distracting options.
+
+..  _pageblindingfunctionmenuoptions-webinfo-example:
+
+Example: Remove some options from the functions menu
+----------------------------------------------------
+
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/page.tsconfig
+
+    mod.web_info.menu.function {
+        # Disable item "Page Tsconfig"
+        TYPO3\CMS\Info\Controller\InfoPageTyposcriptConfigController = 0
```

Comment:
I assume, this would not work anymore since v12, as "TSconfig" is now on second level and not in the "Info" module anymore. But that is something for a separate PR.

Reply:
I did not do any text changes, only indentation and confvals to facilitate backporting.

Reply:
As I said, a separate PR ;-)